internal/feature/features.go

```diff
@@ -122,11 +126,23 @@ func NewContext(ctx context.Context, gate Gate) context.Context {
 	return context.WithValue(ctx, contextKey{}, gate)
 }
 
+// ShowGates returns all the gates that are on by default
+// (if not turned off by the user) or were explicitly set
+// on by the user.
 func ShowGates(ctx context.Context) string {
```

Comment:
❓ Can we name this `ShowFeatures` instead?

```suggestion
// ShowFeatures returns all the features enabled in the Gate contained in ctx.
func ShowFeatures(ctx context.Context) string {
```